be/src/exec/vectorized/hdfs_scanner_text.cpp

```diff
@@ -64,6 +73,10 @@ void HdfsTextScanner::do_close(RuntimeState* runtime_state) noexcept {
 }
 
 Status HdfsTextScanner::do_get_next(RuntimeState* runtime_state, ChunkPtr* chunk) {
+    if (_scanner_params.scan_ranges[0]->offset != 0) {
```

Comment:
If we require `offset = 0`, then we may lose capabilty of reading csv file in chunks, right?

Reply:
yes , i have changed the implementation. skip first record for all the scan range which has non-zero offset value.